packages/terra-arrange/src/Arrange.jsx

```diff
@@ -36,8 +36,20 @@ const propTypes = {
    * The vertical orientation of fitEnd. One of: `center`, `bottom`, `stretch`.
    */
   alignFill: PropTypes.oneOf(alignmentTypes),
+  /**
+   * The max-width to be set on the fitStart element
+   */
+  fitStartMaxWidth: PropTypes.string,
+  /**
+   * The max-width to be set on the fitEnd element
+   */
+  fitEndMaxWidth: PropTypes.string,
 };
 
+const defaultProps = {
+  fitStartMaxWidth: '40%',
```

Comment:
the default should be no max width

Reply:
Without defaulting a max-width; then the start and end containers will not shrink and wrap and will overflow into the fill (center) container and off the screen.  I can set this the '' and then set the start and end containers to flex: 0 1 auto; however, that will cause other issues with other use cases as then the fill content (middle) will squish down the end containers if the middle content is too large.

Reply:
Why can’t the fit container content set their own max width?

Reply:
That doesn't work if consumers want the max width to be a percentage.

Reply:
I am wondering now, though, if it would be better to just have `fitStartAttributes` and `fitEndAttributes` props instead for these types on needs.

Reply:
I agree with @jmsv6d applying the maxWidth and wrapping on the content given doesn't exactly word.  The content can calculate the width of the fit container before applying any text wrapping.
Using firstStart and fitEnd attributes should solve the needs

Reply:
At this point... is it better to just let consumers render their own arrange structure...
```jsx
<Arrange>
  <Arrange.Fit style={{ maxWidth: "40%" }}/>
  <Arrange.Fill/>
</Arrange>
```

Reply:
Also relating to percentage... when would you want that? Generally the fit container is a fixed width and the fill is a width that grows. That means that you  wouldn't want 40%  as that changes with the component parents available width. At this point its sounding more like you want a structural grid than an arrange.

Reply:
I would be okay with that.